### PR TITLE
Modify Smart Seq 2 WDLs with style guide

### PR DESCRIPTION
**Pull request by jsotobroad**

### CODE REVIEW

File: library/tasks/HISAT2.wdl [hunk 1]
```diff
@@ -0,0 +1,313 @@
+task HISAT2PairedEnd {
+  File hisat2_ref
+  File fq1
```
Comment (kbergin) [inline library/tasks/HISAT2.wdl]: I think here and below we would want `fastq1 fastq2` vs `fq1`

File: pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      aligned_bam = HISAT2PairedEnd.output_bam,
+      output_name = quality_control_output_prefix
+  }
+
+  String data_output_prefix = output_name + "_rsem"
+  # TODO should `Trans` be Transcript??
+  call HISAT2.HISAT2RSEM as HISAT2Trans {
+    input:
+      hisat2_ref = hisat2_ref_trans_index,
+      fq1 = fastq1,
```
Comment (kbergin) [inline pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl]: fq1

File: library/tasks/HISAT2.wdl (hunk 1 above)
Comment (jsotobroad) [reply library/tasks/HISAT2.wdl]: good catch!

**Comment by jsotobroad** [comment]
jenkins retest

**Comment by rexwangcc** [comment]
jenkins retest PLZ

**Comment by jsotobroad** [comment]
@jishuxu can you help me fill out the `meta` / `parameter_meta`descriptions.  I put your name everywhere :D

**Comment by jsotobroad** [comment]
jenkins retest

**Comment by jsotobroad** [comment]
retest this please

File: pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    hisat2_ref_name: ""
+    hisat2_ref_trans_name: ""
+    stranded: "Library strand information example values: FR RF NONE"
+    sample_name: ""
+    output_name: ""
+    fastq1: "R1 in paired end reads"
+    fastq2: "R2 in paired end reads"
+  }
+
+  String quality_control_output_basename = output_name + "_qc"
```
Comment (rexwangcc) [inline pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl]: Should there be an empty line between String and call? I can't remember what we decided.

File: library/tasks/HISAT2.wdl [hunk 4]
```diff
@@ -0,0 +1,324 @@
+task HISAT2PairedEnd {
+  File hisat2_ref
+  File fastq1
+  File fastq2
+  String ref_name
+  String output_basename
+  String sample_name
+
+  # runtime optional arguments
+  String? opt_docker
```
Comment (rexwangcc) [inline library/tasks/HISAT2.wdl]: LIKE IT!!!!!!!

File: pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl (hunk 3 above)
Comment (jsotobroad) [reply pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl]: good catch!

**Comment by jsotobroad** [comment]
retest this please

**Pull request merged**